packages/shopping-cart/src/types.ts

```diff
@@ -661,6 +661,7 @@ export interface RequestCartProductExtra extends ResponseCartProductExtra {
 	signup?: boolean;
 	headstart_theme?: string;
 	feature_slug?: string;
+	hosting_intent?: string;
```

Comment:
I realize that none of these other properties have a description but I'd like to change that. Could you add a brief comment above this property to explain what it means?